libs/coin-modules/coin-tezos/src/api/index.ts

```diff
@@ -177,16 +183,16 @@ async function operationsFromHeight(
     accumulator: [],
   };
 
-  let state = await fetchNextPage(address, firstState);
+  let state = await fetchNextPage(address, firstState, order);
   while (state.continueIterations) {
-    state = await fetchNextPage(address, state);
+    state = await fetchNextPage(address, state, order);
   }
   return [state.accumulator, state.nextCursor || ""];
 }
 
 async function operations(
   address: string,
-  pagination: Pagination = { minHeight: 0 },
+  pagination: Pagination = { minHeight: 0, order: "asc" },
 ): Promise<[Operation[], string]> {
-  return operationsFromHeight(address, pagination.minHeight);
+  return operationsFromHeight(address, pagination.minHeight, pagination.order);
```

Comment:
why not remove the operationsFromHeight and call `logic.listOperations` and use the token ?

Reply:
removed